lib/perl/Genome/VariantReporting/Command/Wrappers/ModelReport.pm

```diff
@@ -43,11 +45,58 @@ sub execute {
             plan_file => $model_pair->plan_file($variant_type),
             translations_file => $model_pair->translations_file,
         );
-        Genome::VariantReporting::Command::CreateReport->execute(%params);
+        my $cmd = Genome::VariantReporting::Command::CreateReport->create(%params);
+
+        my $process = $cmd->execute();
+        $process->add_note(
+            header_text => 'creation metadata',
+            body_text => sprintf('%s report created by "genome variant-reporting wrappers model-report --model=%s"',
+                $variant_type, $self->model->id),
+        );
     }
     return 1;
 };
 
+sub get_common_translations {
+    my $self = shift;
+
+    return {
+        sample_name_labels => {
+            $self->get_sample_name_labels,
+        },
+        library_name_labels => {
+            $self->get_library_name_labels('tumor'),
+            $self->get_library_name_labels('normal'),
+        },
+    };
+}
+
+sub get_sample_name_labels {
+    my $self = shift;
+
+    return (
+        $self->model->tumor_sample->name  => sprintf('Tumor(%s)', $self->model->tumor_sample->name),
+        $self->model->normal_sample->name => sprintf('Normal(%s)', $self->model->normal_sample->name),
```

Comment:
This will crash in the single-sample case, since the `normal_sample` will be undefined.